### PR TITLE
fix(ai/serving): rework capablities to avoid endpoint shadowing

### PR DESCRIPTION
**Pull request by XciD**
Fix existing shadowing in /cloud/project/ai/serving/{id} with /cloud/project/ai/serving/capabilities

### CODE REVIEW

File: src/api/cloud/project/ai/capabilities/serving/presetImage/cloud-project-serving-capabilities-stream-region.service.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+angular
+  .module('ovh-api-services')
+  .service('OvhApiCloudProjectAiCapabilitiesServingPresetImage', ($injector) => ({
```
Comment (FredericEspiau) [inline src/api/cloud/project/ai/capabilities/serving/presetImage/cloud-project-serving-capabilities-stream-region.service.js]: Name of the file doesn't match the targeted API

File: src/api/cloud/project/ai/capabilities/serving/flavor/cloud-project-serving-capabilities-stream-region.v6.service.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+angular
+  .module('ovh-api-services')
+  .service('OvhApiCloudProjectAiCapabilitiesServingFlavorV6', ($resource) => {
+    const resource = $resource('/cloud/project/:serviceName/ai/capabilities/serving/flavor', {
```
Comment (FredericEspiau) [inline src/api/cloud/project/ai/capabilities/serving/flavor/cloud-project-serving-capabilities-stream-region.v6.service.js]: Name of the file doesn't match the targeted API

File: src/api/cloud/project/ai/capabilities/serving/presetImage/cloud-project-serving-capabilities-stream-region.v6.service.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+angular
+  .module('ovh-api-services')
+  .service('OvhApiCloudProjectAiCapabilitiesServingPresetImageV6', ($resource) => {
+    const resource = $resource('/cloud/project/:serviceName/ai/capabilities/serving/presetImage', {
```
Comment (FredericEspiau) [inline src/api/cloud/project/ai/capabilities/serving/presetImage/cloud-project-serving-capabilities-stream-region.v6.service.js]: Name of the file doesn't match the targeted API

Comment (XciD) [reply src/api/cloud/project/ai/capabilities/serving/presetImage/cloud-project-serving-capabilities-stream-region.v6.service.js]: nice catch

**Comment by XciD** [comment]
Hello @antleblanc 

I've added your recommendation.

Yes, they are internal for now, should be open in beta next week.

It's possible to merge the PR in order to deploy and test the manager in staging ?

**Pull request merged**